### ISSUE TITLE
quick question about waves/phases.

### ISSUE THREAD

**Issue by Zer0ToHero** [kangarko/CoreArena#199]
hi there, i was wondering, i see it's possible to make it so you only advance to the next wave once all monsters of that wave are dead, but i haven't found anything about making it so there's a timer between each wave starting? like, defeat all monsters on wave 1 and then it waits 10 seconds to advance to wave 2.. is that possible?

**Comment by kangarko** [comment]
Hey,

sure, we got two options for that, simply write them into your arena yml file:

Please note that the first is located

**Comment by Zer0ToHero** [comment]
well, it does answer one question but not precisely what i asked. let me give a scenario;

when i set it to monsters, i kill all monsters then instantly the next wave starts. i dont want the waves on a timer, but i want there to be 10 seconds after each wave where the players can prepare for the next one, rather than the mobs instantly spawning, because what you explained there says that you can have it set to either a timer on each wave or have each wave last until all monsters are killed. not sure if i'm explaining this right.

so basically;

wave 1 starts > kill all monsters > wait 10 seconds > wave 2 starts

**Comment by kangarko** [comment]
No worries, I have coded that in for you! Upgrade the next version version and set the following in your yml file:

![a](https://i.imgur.com/EilrBGL.png)

**Comment by Zer0ToHero** [comment]
awesome! thank you so much, this makes it a ton easier for me, awesome developer :D

**Comment by kangarko** [comment]
If you appreciated this change consider giving a 5* rating on SpigotMC, it will help out so much! Thanks :)
